tutorials/navigation/navigation_using_navigationpaths.rst

```diff
@@ -134,3 +188,65 @@ the default navigation map by setting the target position with ``set_movement_ta
         var new_velocity: Vector3 = global_transform.origin.direction_to(current_path_point) * movement_delta
 
         global_transform.origin = global_transform.origin.move_toward(global_transform.origin + new_velocity, movement_delta)
+
+ .. code-tab:: csharp
+
+    using Godot;
+
+    public partial class MyNode3D : Node3D
+    {
+        private Rid _default3DMapRid;
```

Comment:
Let's add a new line to separate this field from the others, like the GDScript example.

```suggestion
        private Rid _default3DMapRid;

```